### DISCUSSION TITLE
Is there anyway that can we create spLinkDataFrame?

### DISCUSSION THREAD

**Discussion by rapidAmbakar** [moj-analytical-services/splink#828] [Q&A] [answered]
I am trying to understand to internal working of the library, found that we have to specify templated_name and physical_name inorder to create a splinkDataFrame. but can't figure out how to get these params. is there anyway ?

**Comment by RobinL** [comment, marked as answer]
Hello.  This is now possible using the new [register table](https://moj-analytical-services.github.io/splink/linker.html#splink.linker.Linker.register_table) method, which returns a SplinkDataFrame.

Make sure you upgrade to the latest version to use this

**Reply by rapidAmbakar** [reply]
Amazing feature, Thanks 🙌 let me try this out real quick
